controllers/ingress/group_controller.go

```diff
@@ -180,7 +175,11 @@ func (r *groupReconciler) buildAndDeployModel(ctx context.Context, ingGroup ingr
 	}
 	r.logger.Info("successfully deployed model", "ingressGroup", ingGroup.ID)
 	r.secretsManager.MonitorSecrets(ingGroup.ID.String(), secrets)
-	return stack, lb, err
+	if err := r.backendSGProvider.Release(ctx, k8s.ToSliceOfMetaObject(ingress.ExtractIngresses(ingGroup.Members)),
```

Comment:
nit:
if we change the type of ToSliceOfMetaObject  from `[T metav1.Object]` to be `[T metav1.ObjectMetaAccessor]`, and add a GetObjectMeta() Object into `ClassifiedIngress`, we don't need this `ingress.ExtractIngresses`.

This might be better for future changes since we can treat ClassifiedIngress same as Ingress in our code.

Reply:
done